pandas/core/nanops.py

```diff
@@ -460,6 +460,14 @@ def nanmean(values, axis=None, skipna=True, mask=None):
     elif is_float_dtype(dtype):
         dtype_sum = dtype
         dtype_count = dtype
+    elif is_datetime64_dtype(dtype) or is_datetime64tz_dtype(dtype):
```

Comment:
`is_datetime64_any_dtype` I think.

Reply:
Yah, but I'm kind of hoping to get rid of that